user-manager/src/main/resources/application.yaml

```diff
@@ -0,0 +1,11 @@
+spring:
+  flyway:
+    schemas: users
+  jpa:
+    show-sql: true
```

Comment:
move `show-sql: true` to local config since logging of sql queries can significantly reduce performance of app on prod